raspiraw.c

```diff
@@ -211,6 +212,7 @@ static COMMAND_LIST cmdline_commands[] =
 	{ CommandWriteHeaderG,	"-headerg",	"hdg","Sets filename to write the .pgm header to", 0 },
 	{ CommandWriteTimestamps,"-tstamps",	"ts", "Sets filename to write timestamps to", 0 },
 	{ CommandWriteEmpty,	"-empty",	"emp","Write empty output files", 0 },
+	{ CommandDecodeMetadata,	"-metadata",	"meta","Decode register metadata", 0 },
```

Comment:
Personally I'd have gone for -m rather than -meta. The short versions are meant to be as short as practical.